src/ripple/shamap/SHAMapItem.h

```diff
@@ -33,87 +34,58 @@ namespace ripple {
 class SHAMapItem
 {
 private:
-    uint256 mTag;
-    Serializer mData;
+    uint256    tag_;
```

Comment:
I thought we stopped doing the weird internal indentation?  I mean, there are only two items here.  :-)